docs/configuration_yaml_index.md

```diff
@@ -4,6 +4,10 @@ title: "Integration Configuration via YAML"
 
 `configuration.yaml` is a configuration file defined by the user. It is automatically created by Home Assistant on first launch. It defines which components to load.
 
+#### Note about YAML configurations
```

Comment:
There can't a level 4 heading here, as level 1, 2 and 3 are not above it.

Please use a information box instead.

Reply:
I think I've fixed this.  

The Markdown syntax (https://daringfireball.net/projects/markdown/syntax) doesn't seem to have an "information box" that I could find.  I'm using the first answer from Google.  If I'm wrong please point me to the right place for syntax.

Reply:
That would not be a Markdown feature, but a feature of Docusaurus:

<https://v2.docusaurus.io/docs/markdown-features#calloutsadmonitions>

Reply:
I updated to use the Docusaurus info box, but 
`:::info` 
`some stuff`
`:::` 
does not appear to be displayed and showing up like the examples.  Perhaps this is just a Github preview limitation?

Also, I see no mention of Docusaurus at https://developers.home-assistant.io/docs/documenting or the child pages.  Is adding mention of this my next PR?

Reply:
![image](https://user-images.githubusercontent.com/195327/93850895-6314b800-fcaf-11ea-8079-677a3d104398.png)


It looks a bit squashed together now. so a newline might help. Additionally, wrapping the link in a text that is linked looks better in general.

It is correct that it doesn't show up on GitHub, as that only works for GitHub flavored Markdown. See: <https://github.com/home-assistant/developers.home-assistant#preparing-a-local-environment>

Reply:
I wrapped the link and added a newline.  Looks good to me in my new local environment.